src/pages/generic/index.js

```diff
@@ -1,12 +1,14 @@
 import supportedEvents from '../../components/functions/supported.events.vue';
 import supportedFunctions from '../../components/functions/supported.functions.vue';
 import supportedRequests from '../../components/functions/supported.requests.vue';
+import rpcMetadata from '../../components/functions/supported.metadata.vue';
 
 export default {
-    name: 'Supported',
-    pages: [
-        supportedEvents,
-        supportedFunctions,
-        supportedRequests
-    ]
+  name: 'Supported',
+  pages: [
+    supportedEvents,
+    supportedFunctions,
+    supportedRequests,
+    rpcMetadata
+  ]
```

Comment:
I'm not sure if these require their own section. They could pretty much fit one page. It's mostly about how to ask which events, requests and functions are supported on this particular Oskari instance.

Reply:
Moved everything to one page for get supported functionality.